Further fixing (#173)
vippsas/vipps-epayment-api: operations/create.md

| `userFlow`        | Description                                          |
| ----------------- | ---------------------------------------------------- |

> | `WEB_REDIRECT`    | The normal flow for browser-based payment flows. The user will be automatically app-switched to the Vipps MobilePay app if on a mobile device, or be presented with the [Vipps landing page](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/vipps-landing-page/) if not.    |

Reviewer: Switching it to the active voice.

| `NATIVE_REDIRECT` | For automatic app-switch between the merchant's native app and the Vipps MobilePay app. |

> | `PUSH_MESSAGE`    | For payments initiated on a different device than the user's phone. The user gets a push message that opens the payment in the app.Similar to [`skipLandingPage`](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/vipps-landing-page#skip-landing-page) in the [eCom API](https://developer.vippsmobilepay.com/docs/APIs/ecom-api) |

Reviewer: ```suggestion
| `PUSH_MESSAGE`    | For payments initiated on a device other than the user's phone. The user gets a push message that opens the payment in the app. This is similar to [`skipLandingPage`](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/vipps-landing-page#skip-landing-page) in the [eCom API](https://developer.vippsmobilepay.com/docs/APIs/ecom-api). |
```

| `QR`              | Returns a QR code that can be scanned to complete the payment. |

### WEB_REDIRECT

The default flow for:

* Wallet payments:
  Opens the
  [Vipps landing page](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/vipps-landing-page)

>   when the user is on another device than a phone (such as a /laptop computer), and automatically redirects to the

Reviewer: ```suggestion
  when the user is on a device other than a phone (such as a laptop computer), and automatically redirects to the
```

  app when the user is using a phone.
* Card payments:
  Opens the card entry page on both desktop and mobile. More information at
  [Card payments](https://developer.vippsmobilepay.com/docs/vipps-developers/faqs/users-and-payments-faq/#card-payments).

### NATIVE_REDIRECT

Applicable only for `WALLET` payments.

The `redirectUrl` will automatically open the Vipps app on mobile devices, using app-switch.

### PUSH_MESSAGE

Applicable only for `WALLET` payments. 

This will skip the
[Vipps landing page](https://developer.vippsmobilepay.com/docs/vipps-developers/common-topics/vipps-landing-page)
and is only allowed if user is not starting the payment from own device
(e.g., from a Point Of Sale device, vending machines and similar).

If `userFlow` is `PUSH_MESSAGE`, a valid value for `customer.phoneNumber` is required, since there is no
way for the customer to enter the phone number manually.

### QR

Applicable only for `WALLET` payments. 

> For customer-facing screens where payments can be initiated with

Reviewer: ```suggestion
This is for customer-facing screens where payments can be initiated with
```
